ext/iperf3/Kconfig

```diff
@@ -12,7 +12,7 @@ config NRF_IPERF3_PROMPTLESS
 
 config  NRF_IPERF3
 	bool "Iperf3 NRF integration" if !NRF_IPERF3_PROMPTLESS
-	depends on POSIX_API && !PTHREAD_IPC && NEWLIB_LIBC && CJSON_LIB
+	depends on POSIX_API && !POSIX_THREADS && NEWLIB_LIBC && CJSON_LIB
```

Comment:
Remove the dependency to `PTHREAD_IPC` rather than `POSIX_THREADS` and Curl seems to work with `modem_shell` sample. So this line should be:
`depends on POSIX_API && NEWLIB_LIBC && CJSON_LIB`